libraries/src/MVC/Controller/BaseController.php

```diff
@@ -1157,36 +1157,42 @@ public function setRedirect($url, $msg = null, $type = null)
 
 	/**
 	 * Shorten long number to K/M/B/T
-	 * 
+	 *
```

Comment:
This entire numberShorten function is not correct. 

Apart from the fact that this set of abbreviations is not a standard and appears to have been invented
(for example the EU uses Bn its documents) it is hard coded and not translatable

Reply:
I get it. Could you please raise a separate issue for it?